src/ldp/http/metadata/LinkParser.ts

```diff
@@ -2,16 +2,24 @@ import { DataFactory } from 'n3';
 import { getLoggerFor } from '../../../logging/LogUtil';
 import type { HttpRequest } from '../../../server/HttpRequest';
 import { parseParameters, splitAndClean, transformQuotedStrings } from '../../../util/HeaderUtil';
-import { RDF } from '../../../util/Vocabularies';
 import type { RepresentationMetadata } from '../../representation/RepresentationMetadata';
 import { MetadataParser } from './MetadataParser';
 
 /**
- * Parses Link headers with "rel=type" parameters and adds them as RDF.type metadata.
+ * Parses Link headers.
  */
-export class LinkTypeParser extends MetadataParser {
+export class LinkParser extends MetadataParser {
   protected readonly logger = getLoggerFor(this);
 
+  private readonly value: string;
+  private readonly predicate: string;
+
+  public constructor(value: string, predicate: string) {
+    super();
+    this.predicate = predicate;
```

Comment:
Minor optimization: already make a `namedNode` here so the internal metadata store doesn't have to do the conversion.